src/Home/OpenSourceStat.js

```diff
@@ -20,20 +20,20 @@ const OpenSourceStat = styled(({
   indicateColor,
   label,
   dataAgeInDays,
-  className
+  className,
 }) => {
   const disabled = typeof value !== 'number'
   const changeValue = parseFloat(change, 10) > 0 ? `+${change}` : change
 
   return (
     <div title={label} className={`${className} ${disabled ? 'disabled' : ''}`}>
       <OpenSourceStatIcon>
-        <Icon/>
+        <Icon />
       </OpenSourceStatIcon>
-      {disabled ? <div>N/A</div> :
+      {disabled ? <div>N/A</div> : (
         <div>
           <strong>{value}</strong>
-          {dataAgeInDays < 1 ? null :
+          {(dataAgeInDays >= 1) &&
```

Comment:
Should this be `dataAgeInDays > 0` instead? I didn't want to introduce any logic changes in this PR unless the maintainers ask for it.